modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -111,6 +111,7 @@ a-site-pages-variation-with-that-name-already-exists=A site pages variation with
 a-specific-category=A Specific Category
 a-staged-asset-library-cannot-be-connected-to-an-unstaged-site=A staged asset library cannot be connected to an unstaged site.
 a-style-book-with-the-key-x-already-exists=A style book with the key {0} already exists.
+the-property-theme-id-is-missing-from-the-file-x-you-must-specify-the-theme-or-theme-css-client-extension-that-provides-the-style-book-frontend-token-definition=The property themeId is missing from the file 'style-book.json'. You must specify the theme or theme CSS client extension that provides the style book's frontend token definition.
```

Comment:
the-property-theme-id-is-missing-from-the-file-x-you-must-specify-the-theme-or-theme-css-client-extension-that-provides-the-style-book-frontend-token-definition

The key shouldn't have an x since we are not doing a dynamic substitution.

Reply:
Also it should be style-books-frontend-token-definition instead of style-book-frontend-token-definition